actors/test/test_actor.go

```diff
@@ -0,0 +1,48 @@
+package test
+
+import (
+	addr "github.com/filecoin-project/go-address"
+
+	abi "github.com/filecoin-project/specs-actors/actors/abi"
+	builtin "github.com/filecoin-project/specs-actors/actors/builtin"
+	runtime "github.com/filecoin-project/specs-actors/actors/runtime"
+	adt "github.com/filecoin-project/specs-actors/actors/util/adt"
+)
+
+type Actor struct{}
```

Comment:
We'll need a code ID for this actor. Put it in this file rather than alongside the real builtin actors.

Similarly, we also need a table of method numbers so they can be referenced by number.